### DISCUSSION TITLE
📄 New docs released

### DISCUSSION THREAD

**Discussion by mcclowes** [codatio/codat-docs#131] [Announcements]
We've just released our new docs:

https://docs.codat.io/

They are in beta, so you may find some snags in our content.

You can contribute to them [here](https://github.com/codatio/codat-docs), or see the old ones [here](https://codat.readme.io/).

These docs feature an improved organization of content and a more detailed, interactive API reference, driven by our improved [OpenAPI Specification](https://github.com/codatio/oas).

We'd love to hear your opinions, so please let us know how you're finding them in the comments below!

**Comment by anca100** [comment]
The newly created openapi link located here seems to have some issues, this one refers to the absence of version no:  https://apis.guru/?q=accounting
If you select the swagger tool it reports 

Unable to render this definition

The provided definition does not specify a valid version field.

Please indicate a valid Swagger or OpenAPI version field. Supported version fields are swagger: "2.0" and those that match openapi: 3.0.n (for example, openapi: 3.0.0).

Any ideas ?

**Reply by mcclowes** [reply]
Hi @anca100, our OAS file is written as per oAS v3.0.1, so isn't compatible with the tools api guru have chosen to link there.

We do render our OAS in our docs  - e.g https://docs.codat.io/codat-api#/
You can see all of our APIs here: https://docs.codat.io/using-the-api/overview

If you give me a bit more information about what you're trying to achieve I can try and point you in the right direction! 😃

**Reply by anca100** [reply]
Hi

I'm using a proprietary tool to streamline the process of building objects for each API endpoint by automatically discovering the .json url. Although this tool is highly effective with most APIs-Guru listings, it fails with the CODAT listing and your GitHub version. This has led me to question what might be missing from the CODAT listing, especially since the Swagger tool link in your listing also fails with it.

To pinpoint the issue, you could compare the CODAT listing with a working listing, such as the ApiDeck Accounting API.

It's worth noting that I had initially intended to use the Apideck offering, but ultimately chose the CODAT API due to its dynamic capabilities, which align more closely with the project specifications.
